src/models/category.js

```
@@ -0,0 +1,34 @@
+} = require('~/consts/errors')
+
+const categorySchema = new Schema({
+  name: {
+    type: String,
+    required: [true, FIELD_CANNOT_BE_EMPTY('name')],
+    unique: [true, DOCUMENT_ALREADY_EXISTS('name')],
+    minLength: [1, FIELD_CANNOT_BE_SHORTER('name', 1)],
+    maxLength: [150, FIELD_CANNOT_BE_LONGER('name', 150)]
+  },
+  appearance: {
+    icon: {
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)